cmd/csaf_checker/main.go

```diff
@@ -132,10 +135,19 @@ func main() {
 	domains, err := flags.Parse(opts)
 	errCheck(err)
 
-	if len(domains) == 0 {
+	if len(domains) == 0 && !opts.Version {
 		log.Println("No domains given.")
 		return
 	}
+	if opts.Version {
+		version, err := util.GetVersion()
+		if err != nil {
```

Comment:
Couldn't error handling being part of util.GetVersion()? then the returned string could be printed in any case. This would reduce code duplication. (And there should be no case where getVersion fails, see the other remark.)